ui/src/map/layers/alignment/location-track-alignment-layer.ts

```diff
@@ -19,10 +19,28 @@ import { Rectangle } from 'model/geometry';
 import VectorLayer from 'ol/layer/Vector';
 import VectorSource from 'ol/source/Vector';
 import { getLocationTrackMapAlignmentsByTiles } from 'track-layout/layout-map-api';
+import { Stroke, Style } from 'ol/style';
+import mapStyles from 'map/map.module.scss';
 
 let shownLocationTracksCompare = '';
 let newestLayerId = 0;
 
+const highlightedLocationTrackStyle = new Style({
```

Comment:
Pääpointti näissä: kun trackeja ja referencelinejä piirrellään, layerit itse tietää tyylinsä (selected-layerillä se on aina selected, jne). Tällöin ei tarvitse tuolla createAlignmentFeatures-funkkarissa enää iffitellä näitä.